bin/internal/update_dart_sdk.ps1

```diff
@@ -44,7 +44,11 @@ if (Test-Path $dartSdkPath) {
 New-Item $dartSdkPath -force -type directory | Out-Null
 $dartSdkZip = "$cachePath\$dartZipName"
 Import-Module BitsTransfer
-Start-BitsTransfer -Source $dartSdkUrl -Destination $dartSdkZip
+if (-Not (Start-BitsTransfer -Source $dartSdkUrl -Destination $dartSdkZip -errorAction SilentlyContinue)) {
```

Comment:
Hm, Start-BitsTransfer returns a BitsJob object [1]. Not sure what negating that actually means...
I didn't find good documentation for the BitsJobs object, but from stackoverflow it seems like it has a JobState property which might tell you if the transfer had an error.
Alternatively, you could check if Start-BitsTransfer sets $LASTEXITCODE properly.

[1] https://docs.microsoft.com/en-us/powershell/module/bitstransfer/start-bitstransfer?view=win10-ps#outputs

Reply:
Just looked through the bug reports and it seems like no one is reporting failures at this stage currently. Removed this check for now.